source/PrintData.cpp

```diff
@@ -0,0 +1,471 @@
+/* PrintData.cpp
```

Comment:
Can we make this a completely stand-alone executable? (And remove the printing functionality from the main executable completely?)

Reply:
I would tend to agree but shouldn't that be done in another PR?

Reply:
[A reply on a comment in this PR](https://github.com/endless-sky/endless-sky/pull/7106#discussion_r940701155) suggested to me that moving the printing to a stand-alone executable _in this PR_ or _before this PR_ would make the code more clean and maintainable.

Reply:
I'm not sure if a standalone executable makes sense, but either way, with the current way we build every platform this isn't realistic.